### PR TITLE
Updated README.md with an English section

### PR DESCRIPTION
**Pull request by Neel-07**
fixes issue #8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -237,3 +237,169 @@ yarn dev
...
+<details>
+
+<summary> **Test Execution** </summary>
+
+- [Building Test Cases](docs/04-execucao/00-intro.md)
+- [Manual Testing](docs/04-execucao/01-manual.md)
+- [Automated Testing](docs/04-execucao/02-automatizado.md)
+
+</details>
+
```
Comment (vcwild) [inline README.md]: Please, drop the roadmap section as there are no docs for the course in English yet.

Comment (vcwild) [reply README.md]:
```suggestion
```

**Comment by Neel-07** [comment]
@vcwild , removed the roadmap section please review and i have and earnest request can you please add hacktoberfest-accepted 
 label to the issue #8 My pr will get counted. so if you can do this i will be grateful..

**Review by vcwild — APPROVED** [review]
LGTM! :fire: 

Do you need anything else before merging?

**Comment by Neel-07** [comment]
We are good to go, if everything looks fine to you do merge it.

**Pull request merged**